client/cli/src/config.rs

```diff
@@ -166,19 +166,24 @@ pub trait CliConfiguration: Sized {
 		Ok(self.import_params().map(|x| x.database()))
 	}
 
-	/// Get the database configuration.
-	///
-	/// By default this is retrieved from `SharedParams`
-	fn database_config(&self,
```

Comment:
Please keep `self` as the user might need other information in the structopt when they override